src/openapi/openapi.json

```diff
@@ -12912,6 +12912,146 @@
         },
         "x-router-controller": "cors"
       }
+    },
+    "/api/data/sections/{id}": {
+      "parameters": [
+        {
+          "name": "id",
+          "in": "path",
+          "description": "UUID of entity",
+          "required": true,
+          "schema": {
+            "$ref": "#/components/schemas/UUID"
+          }
+        }
+      ],
+      "post": {
+        "tags": [
+          "data"
+        ],
+        "summary": "Update a section",
+        "description": "Returns status of section update",
+        "operationId": "editSection",
+        "requestBody": {
+          "x-name": "payload",
+          "content": {
+            "application/json": {
+              "schema": {
+                "$ref": "#/components/schemas/Section"
+              }
+            }
+          }
+        },
+        "responses": {
+          "200": {
+            "description": "OK",
+            "headers": {
+              "Access-Control-Allow-Origin": {
+                "schema": {
+                  "type": "string"
+                }
+              },
+              "Access-Control-Allow-Methods": {
+                "schema": {
+                  "type": "string"
+                }
+              },
+              "Access-Control-Allow-Headers": {
+                "schema": {
+                  "type": "string"
+                }
+              }
+            },
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/Section"
+                }
+              }
+            }
+          },
+          "500": {
+            "description": "Bad request"
+          }
+        },
+        "security": [
+          {
+            "Cognito_Authorizer": ["openid"]
+          }
+        ],
+        "x-amazon-apigateway-integration": {
+          "uri": "${form_lambda_arn}",
+          "passthroughBehavior": "when_no_match",
+          "httpMethod": "POST",
+          "type": "aws",
+          "requestTemplates": {
+            "application/json": "{\"operation\":\"editSection\",\"params\":{\"payload\": $input.json(\"$\")},\"context\":{\"user_id\": \"$context.authorizer.claims.sub\"}}"
```

Comment:
Digging more into this today, and it just gets more confusing 😂 This configuration is erroring as if the mapping is not being accepted; however, if I don't touch the configuration and just deploy the API stage in APIGateway, it starts working fine. 🤷 This makes me think it has something to do with how the configuration is being converted from this input JSON string to the final value within the mapping template.

Reply:
Still a bit stuck on this. I'm going to have to revisit again later so as not to block merging other PRs. I don't think it has anything directly to do with the code written; however, I want to be sure that the issue doesn't occur in higher environments once merged.

Reply:
This issue should be resolved now. The cause was that the permission being granted between the APIGateway and Lambda were being restricted to just the `GET` request method. This was an issue in a couple other lambda permissions as well. The reason the cause wasn't as obvious and why saving manually within AWS seemed to solve the issue is because AWS was implicitly adding those needed permissions when manually updated in the AWS console.